manifests/d/developersu/ns-usbloader/7.1/developersu.ns-usbloader.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: developersu.ns-usbloader
+PackageVersion: "7.1"
+PackageLocale: en-US
+Publisher: developersu

```

```suggestion
Publisher: '"Dmitry Isaenko"'
```
This should match control panel